server/src/main/java/au/org/aodn/ogcapi/server/core/service/AWSBatchService.java

```diff
@@ -0,0 +1,82 @@
+package au.org.aodn.ogcapi.server.core.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import software.amazon.awssdk.services.batch.BatchClient;
+import software.amazon.awssdk.services.batch.model.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+@Service("AWSBatchService")
```

Comment:
Avoid using @Service and create bean in config, it is easier to control during the test if you need to mock the object